applications/serial_lte_modem/doc/slm_extending.rst

```diff
@@ -7,46 +7,37 @@ Extending the application
    :local:
    :depth: 2
 
-You can extend the serial LTE modem application by adding your own AT commands and custom error handling.
+The AT commands in the Serial LTE modem are implemented with the :ref:`at_cmd_custom_readme` library.
+You can extend the Serial LTE modem application by adding your own AT commands.
 
 Adding AT commands
 ******************
 
-If you want to implement custom AT commands and add them to the serial LTE modem application, see the parser implementation of the provided proprietary AT commands for reference.
-The source files can be found in the :file:`applications/serial_lte_modem/src/` folder.
+If you want to implement a custom AT command in SLM, you will need to follow the instructions in the :ref:`at_cmd_custom_readme` library documentation.
+Setting the AT command filter and callback with the :c:macro:`AT_CMD_CUSTOM` macro is enough for a custom AT command that does not require setup or teardown.
 
-Complete the following steps to add a parser for your own AT commands:
+As an alternative for the :c:macro:`AT_CMD_CUSTOM` macro, SLM defines :c:macro:`SLM_AT_CMD_CUSTOM` macro, which is a wrapper for the :c:macro:`AT_CMD_CUSTOM` macro.
```

Comment:
```suggestion
As an alternative for the :c:macro:`AT_CMD_CUSTOM` macro, SLM defines the :c:macro:`SLM_AT_CMD_CUSTOM` macro, which is a wrapper for the :c:macro:`AT_CMD_CUSTOM` macro.
```

Reply:
I think that it can be left up for the developer. AT_CMD_CUSTOM has an advantage of being nicely documented.

Reply:
Yeah. Anyway I expect anyone that looks into this to look at the code to figure things out.